endpoints.md

```diff
@@ -13,25 +13,29 @@
 * /api/core/metadataschemas
 * /api/core/epersons
 * /api/core/groups
-* /api/core/{model}/search
-* [/api/discover/browses](browses.md)
-
-## Endpoints Under Development
-
-* [/api/discover/search](search-endpoint.md)
+* [/api/core/{model}/search](search-rels.md)
 * [/api/config/submissiondefinitions](submissiondefinitions.md)
 * [/api/config/submissionsections](submissionsections.md)
 * [/api/config/submissionforms](submissionforms.md)
+* [/api/discover/browses](browses.md)
+* [/api/integration/authorities](authorities.md)
+
+## Endpoints Under Development/Discussion
+* [/api/discover/search](search-endpoint.md)
+* [/api/config/submissionuploads](submissionforms.md)
 * [/api/integration/authorities](authorities.md)
 * [/api/submission/workspaceitems](workspaceitems.md)
+* /api/authn/login
+* /api/authn/logout
+* /api/authn/status
+* /api/authz/resourcepolicies
```

Comment:
Tiny thing (and I realize this isn't part of this PR, technically):  I wonder if we should remove the "authn" prefixes here, and just have endpoints like `/api/login`, `/api/logout`, etc.   These are such core needs, they might best be top level endpoints.   For Resource Policies, we could leave it as either `authz/` or simplify it to just `auth/`   (Overall, I'm not sure the distinction of authN vs authZ here is very meaningful to most people, as the two are rather tightly connected.)